server/players.py

```diff
@@ -76,6 +79,39 @@ def __init__(
         self._game = lambda: None
         self._game_connection = lambda: None
 
+    @property
+    def global_rating(self):
+        return self._global_rating
+
+    @global_rating.setter
+    def global_rating(self, value: Rating):
+        if isinstance(value, Rating):
+            self._global_rating = (value.mu, value.sigma)
+        else:
+            self._global_rating = value
```

Comment:
if `value` is `None`, you may achieve bad behavior.

Reply:
This is the original behaviour and it was not my intention to change it. It's not set to `None` internally and if there's an inconsistency with how other code uses it, then you can show and fix it.

Reply:
Maybe it would be a good opportunity to add `assert value is not None`

Reply:
I don't see benefit from moving code from one place to another, if we can make it better.

Reply:
@norraxx Because it's a lot of extra work and a lot higher chance that you mess something up on accident. I'd be happy if you wanted to make a PR though :)